### ISSUE TITLE
Missing dependencies

### ISSUE THREAD

**Issue by remixer-dec** [noamgat/lm-format-enforcer#79]
Hi, after installing lm-format-enforcer from pypi in a clean container, it fails to import some modules.
Running `from lmformatenforcer import CharacterLevelParser, JsonSchemaParser`
```log
  File "/usr/local/lib/python3.10/dist-packages/lmformatenforcer/external/jsonschemaobjectutil.py", line 29, in <module>
    from packaging import version
ModuleNotFoundError: No module named 'packaging'
```

```log
  File "/usr/local/lib/python3.10/dist-packages/lmformatenforcer/external/jsonschemaobjectutil.py", line 69, in <module>
    from yaml import CSafeLoader as SafeLoader
ModuleNotFoundError: No module named 'yaml'
```
installing `packaging` and `PyYAML` fixes this.

**Comment by remixer-dec** [comment]
I don't think it has anything to do with poetry. These dependencies are not present in `pyproject.toml` and both pip and poetry do not install them.

**Comment by noamgat** [comment]
Fixed in v0.9.1
